<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rabbit Man</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/new/prefix1/main.png</normaloff>:/new/prefix1/main.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="margin">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Droid Sans Japanese</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>ファイル(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>編集(&amp;E)</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDel"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
   </widget>
   <widget class="QMenu" name="menu_B">
    <property name="title">
     <string>ビルド(&amp;B)</string>
    </property>
    <addaction name="actionComp"/>
    <addaction name="actionMake"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>ヘルプ(&amp;H)</string>
    </property>
    <addaction name="actionAboutQt"/>
    <addaction name="actionVer"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>ツール(&amp;T)</string>
    </property>
    <addaction name="actionFontEditor"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_B"/>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/new/prefix1/qt-logo.png</normaloff>:/new/prefix1/qt-logo.png</iconset>
   </property>
   <property name="text">
    <string>Qtについて</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionVer">
   <property name="text">
    <string>バージョン情報</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>終了(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/new/prefix1/two-docs.gif</normaloff>:/new/prefix1/two-docs.gif</iconset>
   </property>
   <property name="text">
    <string>ファイルを開く(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/new/prefix1/text-bigger.gif</normaloff>:/new/prefix1/text-bigger.gif</iconset>
   </property>
   <property name="text">
    <string>新しいファイル(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/new/prefix1/diskette.gif</normaloff>:/new/prefix1/diskette.gif</iconset>
   </property>
   <property name="text">
    <string>名前を付けてファイルを保存(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/new/prefix1/checkmark.gif</normaloff>:/new/prefix1/checkmark.gif</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>切り取り</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>コピー</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>貼り付け</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="text">
    <string>削除</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionComp">
   <property name="text">
    <string>コンパイル</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionFontEditor">
   <property name="text">
    <string>フォントの設定</string>
   </property>
  </action>
  <action name="actionMake">
   <property name="text">
    <string>メイク</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>ズームイン</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>ズームアウト</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+.</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
